src/Kodify/BlogBundle/Resources/views/Post/view.html.twig

```diff
@@ -9,5 +9,34 @@
         <div class="panel-body">
             {{ post.content }}
         </div>
+        <div class="panel-footer">
+            {% include 'KodifyBlogBundle:Post:Component/rating.html.twig' %}
+        </div>
     </div>
-{% endblock %}
\ No newline at end of file
+
+    <section id="comments">
+        <header>
+            <h1>Comments</h1>
+        </header>
+        <ul class="list-unstyled">
+            {% for comment in post.comments %}
+                <li class="comment">
+                    <div>
+                        <span class="comment-author">{{ comment.author.name }}</span>
+                    </div>
+                    <div>
+                        <span class="comment-text">{{ comment.text }}</span>
+                    </div>
+                    <div class="comment-date">
+                        {{ comment.createdAt|date('d/m/Y H:i') }}
+                    </div>
+                </li>
+            {% else %}
+                <li class="no-comments">There are no comments</li>
+            {% endfor %}
```

Comment:
Right now here you show up to 10 comments. How would you do an option to see more of them and what would you take into account? (There's no need to do it programmatically, just a quick explanation :smile:)